package.json

```diff
@@ -18,6 +18,9 @@
   "bugs": {
     "url": "https://github.com/cerner/eslint-config-terra/issues"
   },
+  "engines": {
+    "node": "^8.10.0 || ^10.13.0 || >=11.10.1"
```

Comment:
Set only 8.10.0 as the minimum version or also include minimum version for 10 and 11 to [match](https://github.com/eslint/eslint/blob/4e142ea411dfb692b6e2a69cd5f1204ade4dd58a/package.json#L143) what eslint sets?

Reply:
Sooo... are we going to release this as a MVB or as a bug fix?

Reply:
I believe this would only give them a warning during npm install if the min version is not met as seen [here](https://github.com/eslint/eslint/issues/12493#issuecomment-546928948). So I would think just as a bug fix. 

However, I don't get the warning during npm install when I default node to 8.9.2. I don't know if it is because I also have 10.17 installed, it's just that it is not using it.

Reply:
The engine field is ["advisory only and will only produce warnings when your package is installed as a dependency"](https://docs.npmjs.com/files/package.json#engines).